tools/pot/tests/test_scales.py

```diff
@@ -163,8 +166,8 @@ def test_weights_scales(tmp_path, models):
         ref_min, ref_max = ref_weights[fq_name]['low_level'], ref_weights[fq_name]['high_level']
         assert_flag = False
 
-        if np.array_equal(ref_min, item_min) and \
-                np.array_equal(ref_max, item_max):
+        if np.allclose(ref_min, item_min, atol=eps) and \
+                np.allclose(ref_max, item_max, atol=eps):
```

Comment:
@AlexKoff88, I've updated the way values compare to each other in `test_weights_scales`

Reply:
I wonder where the `eps` value is coming from? can we make it lower?

Reply:
I just took this value from `test_fake_quantize_configurations`. I think we could try 1e-6 value in this tests

Reply:
I am for a lower value. It can prevent from having problems in the future.